### ISSUE TITLE
Is this ready for testing?

### ISSUE THREAD

**Issue by vdrover** [colinbendell/homebridge-blink-for-home#1]
Hi there, 

Some of us are very interested to help you test this. Can you share your plans?

**Comment by colinbendell** [comment]
Howdy! I love that this was opened as an issue :)

I think this is relatively ready for a 0.1 release. I finally cracked the LiveView code, but need some time to write a new RTSP stack because Blink has a broken implementation. The current state can be tested, I've got additional refactoring and docs to be pushed shortly. Then on to fixing liveview

For those interested, the Blink RTSP is wrapped in a TLS socket, but just for transport. The protocol is just relayed and so ffmpeg and openRTSP fall over because the `m=` line doesn't include `/TCP`. Blink App clearly fails gracefully, but for the camera to operate via ffmpeg and homekit, we need to do some intercept and proxy the RTSP comms. It's just work from here on in.
